electrumx/server/session.py

```diff
@@ -1468,6 +1474,51 @@ async def transaction_broadcast(self, raw_tx):
             self.logger.info(f'sent tx: {hex_hash}')
             return hex_hash
 
+    async def package_broadcast(self, tx_package: List[str], verbose: bool = False) -> dict:
+        """Broadcast a package of raw transactions to the network (submitpackage).
+        The package must consist of a child with its parents,
+        and none of the parents may depend on one another.
+
+        raw_txs: a list of raw transactions as hexadecimal strings"""
+        self.bump_cost(0.25 + sum(len(tx) / 5000 for tx in tx_package))
+        try:
+            txids = [double_sha256(bytes.fromhex(tx)).hex() for tx in tx_package]
+        except ValueError:
+            self.logger.info(f"error calculating txids: {traceback.format_exc()}")
```

Comment:
no need to import traceback
```suggestion
        except ValueError as e:
            self.logger.info(f"error calculating txids", exc_info=e)
```

-----

here is some boilerplate:
```py
import traceback
import logging
import sys

logger = logging.getLogger(__name__)
logger.setLevel(logging.DEBUG)
console_stderr_handler = logging.StreamHandler(sys.stderr)
console_stderr_handler.setLevel(logging.DEBUG)
logger.addHandler(console_stderr_handler)
```

and now compare (in a fresh local python interpreter):
```py
try:
    txids = [double_sha256(bytes.fromhex(tx)).hex() for tx in tx_package]
except Exception:
    logger.info(f"error calculating txids: {traceback.format_exc()}")
```
and
```py
try:
    txids = [double_sha256(bytes.fromhex(tx)).hex() for tx in tx_package]
except Exception as e:
    logger.info(f"error calculating txids", exc_info=e)
```
and
```py
try:
    txids = [double_sha256(bytes.fromhex(tx)).hex() for tx in tx_package]
except Exception:
    logger.info(f"error calculating txids", exc_info=True)
```
see https://docs.python.org/3/library/logging.html#logging.Logger.debug